linea-besu-package/linea-besu/Dockerfile

```diff
@@ -26,6 +26,7 @@ USER besu
 WORKDIR /opt/besu
 
 COPY --chown=besu:besu besu /opt/besu/
+RUN sed -i '/^CLASSPATH/c\CLASSPATH=/opt/besu/lib/\*\:/opt/besu/plugins/\*' /opt/besu/bin/besu
```

Comment:
What's the context of adding this line here? Also does it mean we could remove the same line [here](https://github.com/Consensys/linea-monorepo/blob/51a0c52d3386481e7b4c99f613cb1f3cd5cb6900/docker/compose-spec-l2-services.yml#L547) later on?